### DISCUSSION TITLE
What is the best autotune in Ardour

### DISCUSSION THREAD

**Discussion by Franklin-Russell** [community/community#21291] [Apps, API and Webhooks]
I am using autotune and notice some artifacts.
Is autotune lv2 the best?  Is there a stereo version?  Thanks Mahalo!

**Comment by pidoshva** [comment]
**Here are some of the best ones I could find:**
- x42-Autotune:
[https://x42-plugins.com/x42/x42-autotune](https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbjBobER0LUFvWE5zcDRqeHd1TF8tenUzQTBtQXxBQ3Jtc0tsSlRQOUdkQkQwRkpBbXJsY0E4dVhaWWVGdk1qSzRHQ0FJa0J3Y3dkTkplMk42aGNac2JmSGhBV01mTGx1cVFVOGtYSGdPS3Zjb1dJRjdvb3o4S1ZGQTdxS1BkWjIyOWFzTXV1REh6ZW5OclNSOVpxZw&q=https%3A%2F%2Fx42-plugins.com%2Fx42%2Fx42-autotune&v=ttXCsIb_2L8)

- Autotalent:
[http://tombaran.info/autotalent.html](https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqay00QUt0TVRVOWZjT3ZtM3VUNlpPRG5meW1RZ3xBQ3Jtc0tuamp3anI5eU4xT2dralRVTVpsZEE3YnZmakVRd1Bpa3VrMWFkREUwYW1nbklWU2tGSWVRTG1yWTkzR25yVjJSTEZrZTlYdjQ5Y2Rjc2Y5eWNOVjJXV04wNUVxdVlVWUdrOTlpNXkwRDE4SWE4VWc0bw&q=http%3A%2F%2Ftombaran.info%2Fautotalent.html&v=ttXCsIb_2L8)

- TalentedHack (improved Autotalent):
[https://github.com/jeremysalwen/Talen...](https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbGs4bkVaRmsybmxMMERleWpSRjlYVGxUc0d4QXxBQ3Jtc0tsWVZRMk1VczFVUjM2VmxxVllqRzBELTFUV09yUkVOMlhxcjJoNlZWWkY1YzRIcFBndXc3N0JiaHM3cjZ5VUhQT0pZeDV0WF83aXpIR21kME1BY1U1cWJHVDJ0OWIxLWZ4VHIzYzVEeUNUYmtBWjBjTQ&q=https%3A%2F%2Fgithub.com%2Fjeremysalwen%2FTalentedHack&v=ttXCsIb_2L8)
